stix_shifter_modules/trustar/stix_translation/json/to_stix_map.json

```diff
@@ -0,0 +1,251 @@
+{
+  "IP": {
+    "key": "ipv4-addr.value",
+    "object": "ipv4-addr"
+  },
+  "MALWARE": {
+    "key": "x-trustar-malware.value",
+    "object": "x-trustar-malware"
+  },
+  "URL": {
+    "key": "url.value",
+    "object": "url"
+  },
+  "CVE": {
+    "key": "vulnerability.id",
+    "object": "vulnerability"
+  },
+  "EMAIL_ADDRESS": {
+    "key": "email-addr.value",
+    "object": "email-addr"
+  },
+  "REGISTRY_KEY": {
+    "key": "windows-registry-key.id",
+    "object": "registry-key"
+  },
+  "MD5": {
+    "key": "file.hashes.MD5",
+    "object": "file"
+  },
+  "SHA1": {
+    "key": "file.hashes.SHA1",
+    "object": "file"
+  },
+  "SHA256": {
+    "key": "file.hashes.SHA-256",
+    "object": "file"
+  },
+  "SOFTWARE": [{
+      "key": "file.id",
```

Comment:
there's no id property in file and software objects. this field can be mapped to software name

Reply:
Done the changes

Reply:
software object still has id property that needs to be removed

Reply:
https://github.com/sathya573/stix-shifter/blob/trustar-CodeReview/stix_shifter_modules/trustar/stix_translation/json/to_stix_map.json

Done the changes.

Reply:
still there!
```
{
       "key": "software.id",
       "object": "software",
       "references": "file"
     }
```

Reply:
update the mapping.
https://github.com/sathya573/stix-shifter/blob/trustar-CodeReview/stix_shifter_modules/trustar/stix_translation/json/to_stix_map.json#L43